### PR TITLE
[chore](fe)correct some doc errors

### PR DESCRIPTION
**Pull request by tudouzhao**
## Proposed changes

Issue Number: close #xxx

edit_log rolls at threshold of 5w logs, not 10w.
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by nimuyuhan — APPROVED** [review]
LGMT

**Review by luozenglin — APPROVED** [review]
LGTM

**Comment by tudouzhao** [comment]
run buildall

**Comment by tudouzhao** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

File: docs/en/community/design/metadata-design.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ The data flow of metadata is as follows:
...
 
-3. The number of log entries of the leader node reaches the threshold (default 10w) and meets the checkpoint thread execution cycle (default 60 seconds). Checkpoint reads existing image files and subsequent logs and replays a new mirror copy of metadata in memory. The copy is then written to disk to form a new image. The reason for this is to regenerate a mirror copy instead of writing an existing image to an image, mainly considering that the write operation will be blocked during writing the image plus read lock. So every checkpoint takes up twice as much memory space.
+3. The number of log entries of the leader node reaches the threshold (default 5w) and meets the checkpoint thread execution cycle (default 60 seconds). Checkpoint reads existing image files and subsequent logs and replays a new mirror copy of metadata in memory. The copy is then written to disk to form a new image. The reason for this is to regenerate a mirror copy instead of writing an existing image to an image, mainly considering that the write operation will be blocked during writing the image plus read lock. So every checkpoint takes up twice as much memory space.
```
Comment (dataroaring) [inline docs/en/community/design/metadata-design.md:76]: It seems that there is no threshold on log entries.

**Pull request closed without merging**